src/pages/Admin/AdminVenueRoomDetails.tsx

```diff
@@ -122,13 +122,28 @@ export const AdminVenueRoomDetails = ({
                       name={"toggle-" + index}
                       checked={room.isEnabled}
                       onClick={() => {
-                        updateRoom(!room.isEnabled);
+                        updateRoom(!room.isEnabled, "isEnabled");
                       }}
                     />
                     <span className="slider round"></span>
                   </label>
                   <div>Turn room {room.isEnabled ? "Off" : "On"}</div>
                 </div>
+                <div className="toggle-room">
+                  <label id={"toggle2-" + index} className="switch">
+                    <input
+                      type="checkbox"
+                      id={"toggle2-" + index}
+                      name={"toggle2-" + index}
+                      checked={room.isLabeled}
+                      onClick={() => {
+                        updateRoom(!room.isLabeled, "isLabeled");
+                      }}
+                    />
+                    <span className="slider round"></span>
+                  </label>
+                  <div>Label: {room.isLabeled ? "On" : "Off"}</div>
+                </div>
```

Comment:
I realise that this is following the pattern of the code above it, but it's quite old/legacy.  Is it possible to use `src/components/atoms/Toggler/Toggler.tsx` here instead? That would be ideal.

I don't think we should need the `id`s here, since we nest the `input` within the `label` as per:

- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label
  - > Alternatively, you can nest the `<input>` directly inside the `<label>`, in which case the `for` and `id` attributes are not needed because the association is implicit

Reply:
I ended up removing this toggler entirely, as we don't need this to be exposed on the main view. It's more of an advanced setting, so can be 'hidden away' only in the main 'edit room' dialog.